rules/windows/process_creation/proc_creation_win_susp_use_of_sqlps_bin.yml

```diff
@@ -21,14 +21,13 @@ logsource:
     product: windows
 detection:
     selection_1:
-        Image|endswith: '\sqlps.exe'
+        - Image|endswith: '\sqlps.exe'
+        - OriginalFileName: 'sqlps.exe'
     selection_2:
         ParentImage|endswith: '\sqlps.exe'
-    selection_3:
-        OriginalFileName: '\sqlps.exe'
     filter:
         ParentImage|endswith: '\sqlagent.exe'
-    condition: 1 of selection_* and not filter
+    condition: (selection_1 and not filter) or selection_2
```

Comment:
just to make it easier to understand when reading
```yaml
detection:
    selection_parent:
        ParentImage|endswith: '\sqlps.exe'
    selection_image:
        - Image|endswith: '\sqlps.exe'
        - OriginalFileName: 'sqlps.exe'
    filter_image:
        ParentImage|endswith: '\sqlagent.exe'
    condition: selection_parent or (selection_image and not filter_image)
```

Reply:
Done